server/config/express.js

```diff
@@ -17,17 +16,36 @@ const express = require('express'),
 	xml2js = require('xml2js').parseString,
 	axios = require('axios'),
 	authMiddleware = require('../src/middlewares/auth'),
-	{ errors, isCelebrate } = require('celebrate');
+	{ errors, isCelebrate } = require('celebrate'),
+	bunyan = require('bunyan'),
+	seq = require('bunyan-seq'),
+	bunyanMiddleware = require('bunyan-middleware');
 
 const swaggerSpec = swaggerJSDoc({
 	swaggerDefinition: swaggerDefinition,
 	apis: ['./docs/**/*.yaml'],
 });
 
 module.exports = (app, config) => {
-
-	if (app.get('env') === 'development') {
-		app.use(logger('dev'));
+	const logger = bunyan.createLogger({
+		Application: 'Website',
+		Environment: app.get('env'),
+		name: 'Website',
+		streams: [
+			{
+				stream: process.stdout,
+				level: 'warn',
+			},
+			seq.createStream({
+				serverUrl: process.env.SEQ_ADDRESS,
+				apiKey: process.env.SEQ_TOKEN,
+				level: 'info',
+			})
```

Comment:
Also, what happens if people don't have these defined? Is Seq now a requirement for development or does this gracefully handle it?